ADALiOS/ADALiOS/ADAuthenticationRequest+AcquireToken.m

```diff
@@ -220,14 +220,12 @@ - (void)requestToken:(ADAuthenticationCallback)completionBlock
          }
          else
          {
-#if BROKER_ENABLED
```

Comment:
I added BROKER_ENABLED in a previous change. Those ifdefs weren't correct so I just removed them.